lib/wordsmith/client.rb

```diff
@@ -6,49 +6,55 @@ module Wordsmith
   class Client
 
     def get(uri)
-      response = connection.get uri
-      parse_response response
+      response = connection.get(uri)
+      parse_response(response)
     end
 
-    def post(uri, data)
-      response = connection.post uri, {data: data}.to_json
-      parse_response response
+    def post(uri, data, proofread)
+      response = connection.post(uri, {data: data, proofread: proofread}.to_json)
+      parse_response(response)
     end
 
     private
 
     def connection
       return @_connection if connection_valid?
-      @_connection = Faraday.new(url: Wordsmith.configuration.url, headers: {
-        'Content-Type' => 'application/json',
-        'Authorization' => "Bearer #{Wordsmith.configuration.token}",
-        'User-Agent' => Wordsmith.configuration.user_agent
-      })
+      @_connection = initialize_connection
+    end
+
+    def initialize_connection
+      Faraday.new(
+        url: Wordsmith.configuration.url,
+        headers: {
+          'Content-Type' => 'application/json',
+          'Authorization' => "Bearer #{Wordsmith.configuration.token}",
+          'User-Agent' => Wordsmith.configuration.user_agent
+        }
+      )
     end
 
     def connection_valid?
       return false unless @_connection
       url = @_connection.url_prefix.to_s
       authorization = @_connection.headers['Authorization']
 
-      url == Wordsmith.configuration.url &&
-        authorization == "Bearer #{Wordsmith.configuration.token}"
+      url == Wordsmith.configuration.url && valid_token?(authorization)
+    end
+
+    def valid_token?(authorization)
+      authorization == "Bearer #{Wordsmith.configuration.token}"
     end
 
     def parse_response(response)
       body = JSON.parse(response.body)
-      Hashie.symbolize_keys! body
+      Hashie.symbolize_keys!(body)
       case response.status
-      when 200, 201
-        return body[:data]
-      when 400
-        fail %Q(Bad Request: "#{body[:errors]}")
-      when 401
-        fail 'API authorization error'
-      when 429
-        fail body[:error]
-      else
-        fail 'API error'
+      when 200, 201 then body[:data]
+      when 400 then fail %Q(Bad Request: "#{body[:errors]}")
```

Comment:
(another obvious exception is `Minitest` `assert` and `refute` methods, so maybe Rubocop has some leeway in that rule?)